src/Tour.js

```diff
@@ -569,6 +572,10 @@ const CN = {
     base: 'reactour__helper',
     isOpen: 'reactour__helper--is-open',
   },
+  dot: {
+    base: 'reactour__dot',
+    active: 'reactour__dot--active',
```

Comment:
@elrumordelaluz 

I implemented your suggestions. Sorry, I wasn't familiar enough with your codebase.

Also, wondering if we really need a `base` class here. Or if we should call this `reactour__dot--is-active` to be more consistent with the naming?

Reply:
no problem at all @nelsonwong1012 
The _base_ `class` could be useful at the moment of customizing, like your needs for the _active_ one but for the _inactive_ ones.

LGFM `reactour__dot--is-active`